ui/cypress/integration/node/nodetabs.spec.js

```diff
@@ -0,0 +1,121 @@
+beforeEach(() => {
+  cy.setupMocks();
+  cy.login();
+});
+
+// Navigation tests
+describe('Overview tab in master-0 node page', () => {
+  beforeEach(() => {
+    cy.server();
+    cy.visit('/nodes/master-0/overview');
+    cy.route('api/kubernetes/api/v1/nodes', 'fixture:kubernetes/nodes.json');
+    cy.window()
+      .its('__history__')
+      .then((history) => {
+        cy.stub(history, 'push').as('historyPush');
+      });
+  });
+
+  const tabs = ['alerts', 'metrics', 'volumes', 'pods', 'details'];
+  tabs.map((tab) => {
```

Comment:
This is not specific to the `overview` tab, so maybe it makes sense get this test in a more generic context (e.g. `Node page tabs`), and make sure that we can reach any tab from any other tab (and also check that clicking the current tab doesn't do anything).